index.js

```diff
@@ -32,4 +65,8 @@ const AWS = "AWS";
       JSON.stringify({ total: result.total })
     );
   }
+  console.log("CSPM/CIEM Asset Units: " + CSPM_UNITS);
+  console.log("CWPP Asset Units: " + CWPP_UNITS);
+  console.log("Total Asset Units: " + (CWPP_UNITS + CSPM_UNITS));
```

Comment:
We should probably calculate the monthly usage and map them to a tier. The tier information can be retrieved from @PierreLiddle

Reply:
@subeshb1 we could add this as a future improvement? Pierre's happy about the current format

Reply:
I think we should double-check this. The reason we want to display the units is to map the customer to a tier so they can estimate how much it's going to charge them.

Feel free to resolve this, if we don't want to take it as an improvement in the future.

Reply:
I'm just a bit hesitant because if we say what their tier is then it may be misleading because we're not collecting CDR events. It also means that if we change that tiers then we'll also need to update it here. I'll run it past Pierre

Reply:
@subeshb1 yeah I just talked to Pierre let's leave out tier for now

Reply:
Thanks @edmundcong